### ISSUE TITLE
Ultra-Spectropic Goggles/Gas Cloud Tracking

### ISSUE THREAD

**Issue by Myllz** [SunwellWoW/Sunwell-TBC-Bugtracker#3832]
Description: When wearing goggles to show you gas clouds on the mini-map, it turns off any other tracking.

How it works: The only way you can track gas clouds by wearing the goggles is if you have no other tracking on.

How it should work: You should be able to track other items (minerals, herbs, etc.) while wearing the goggles and still being able to see gas nodes.

Source (you should point out proofs of your report, please give us some source): 

Wowhead comments:
![image](https://user-images.githubusercontent.com/669814/50427626-5dd8ce00-087b-11e9-9946-147475f67184.png)
![image](https://user-images.githubusercontent.com/669814/50427629-66c99f80-087b-11e9-83ae-d7aa6948e7ce.png)

**Comment by mintnb** [comment]
This is correct. I remember when I had to hover over the dots on the minimap to see if it is a cloud or a mining node.
